tests/test.py

```diff
@@ -564,7 +568,7 @@ def execute_prepared_statement(self, query, args=None):
                 cursorclass=pymysql.cursors.DictCursor,
                 **pymysql_tls_args)) as connection:
             with connection.cursor() as cursor:
-                cursor.execute("PREPARE test_statement FROM '{}'".format(query))
+                cursor.execute("PREPARE test_statement FROM {}".format(str(sa.literal(query).compile(dialect=db_dialect, compile_kwargs={"literal_binds": True}))))
```

Comment:
Can you please explain what does it mean? How query will be changed?

Reply:
in mysql sql query should be string literal or some variable that contains query to execute. Literal should be passed in the same manner as other literals in quotes and may have other quotes and inner literals (for exampe `select * from table where a='123'). Here I use sqlalchemy python library that can create sql queries for specified database (if they have some differences then it will create different sql queries for each dialect) with correct escaping quotes inside string literal. 
As default it return placeholder for value that should be replaced with value before sending to database (sqlalchemy uses ":param1", ":param2" placeholders and then replace with values) but here we need binded values instead placeholder because then this query will be sent without sqlalchemy usage